spnav_controller:
  type: "tiago_spnav_teleop/SpnavControllerPlugin"
  joint_names: ["arm_1_joint", "arm_2_joint", "arm_3_joint", "arm_4_joint", "arm_5_joint", "arm_6_joint", "arm_7_joint"]
